icechunk-python/src/config.rs

```diff
@@ -492,15 +492,15 @@ impl From<&PyCompressionConfig> for CompressionConfig {
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct PyCachingConfig {
     #[pyo3(get, set)]
-    pub snapshots_cache_size: u16,
+    pub snapshots_nodes: u64,
     #[pyo3(get, set)]
-    pub manifests_cache_size: u16,
+    pub manifests_refs: u64,
     #[pyo3(get, set)]
-    pub transactions_cache_size: u16,
+    pub transactions_records: u64,
     #[pyo3(get, set)]
-    pub attributes_cache_size: u16,
+    pub attributes_bytes: u64,
     #[pyo3(get, set)]
-    pub chunks_cache_size: u16,
+    pub chunks_bytes: u64,
```

Comment:
Why switch from `u16` to `u64`? I guess 2**16 or 2**32 bytes is too small? A "docstring" comment saying we expect the user to set "reasonable values" would be useful